recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script:
+    - if note exists %SCRIPTS% mkdir %SCRIPTS%
+    - xcopy /v %RECIPE_DIR%\\setup_x64.bat %SCRIPTS%\\setup_x64.bat
+    - xcopy /v %RECIPE_DIR%\\vs2008 %SCRIPTS%
```

Comment:
As things got moved around in interim, will need to change ``%RECIPE_DIR%` -> `%RECIPE_DIR%\\vs2008_patch`.